### DISCUSSION TITLE
Can we have multiple subdomains?

### DISCUSSION THREAD

**Discussion by daniel4-scratch** [is-a-dev/register#1832] [Q&A] [answered]
Can we make more than one subdomain? Or is it recommended not to.

**Comment by MaskDuck** [comment, marked as answer]
you can register ∞ subdomains, as long as it isnt violate our rules.

**Reply by devashish2024** [reply]
Hello there, can I get cutyoururl.is-a.dev?  Ihave  ashishagarwal.is-a.dev but I want one for my project too. Plus, is there any kind of django/flask/vercel/python hosting smth?
